### PR TITLE
Udev cleanup follow-ups

### PR DESCRIPTION
**Pull request by keszybz**
Tiny follow-ups for #10861.

### CODE REVIEW

File: src/libsystemd/sd-device/sd-device.c [hunk 1]
```diff
@@ -793,9 +793,9 @@ _public_ int sd_device_get_subsystem(sd_device *device, const char **ret) {
                 else if (path_startswith(device->devpath, "/module/"))
                         r = device_set_subsystem(device, "module");
                 else if (!(drivers = strstr(syspath, "/drivers/")) &&
-                         (path_startswith(device->devpath, "/subsystem/") ||
-                          path_startswith(device->devpath, "/class/") ||
-                          path_startswith(device->devpath, "/bus/")))
+                         PATH_STARTSWITH_SET(device->devpath, "/subsystem/",
+                                                              "/class/",
+                                                              "/bus/"))
```
Comment (yuwata) [inline src/libsystemd/sd-device/sd-device.c:798]: Why do you keep the linebreaks?

File: src/basic/path-util.h [hunk 2]
```diff
@@ -8,6 +8,7 @@
 #include "macro.h"
 #include "string-util.h"
 #include "time-util.h"
+#include "strv.h"
```
Comment (poettering) [inline src/basic/path-util.h:10]: not that it matters, but this is not ordered alphabetically...

Comment (keszybz) [reply src/basic/path-util.h:10]: Updated.

File: src/libsystemd/sd-device/sd-device.c (hunk 1 above)
Comment (keszybz) [reply src/libsystemd/sd-device/sd-device.c:798]: I looked cluttered when it was all on one line.

**Pull request merged**